src/client/rsg-components/Preview/Preview.tsx

```diff
@@ -88,7 +88,7 @@ export default class Preview extends Component<PreviewProps, PreviewState> {
 			try {
 				ReactDOM.render(wrappedComponent, this.mountNode);
 			} catch (err) {
-				this.handleError(err);
+				this.handleError(err as Error);
```

Comment:
This would be better than silencing type checker:

```suggestion
        if (err instanceof Error) {
				  this.handleError(err as Error);
				}
```

Same in other places.

Reply:
fixed